design/capm3-remediation-controller-proposal.md

```diff
@@ -143,15 +148,16 @@ metadata:
   namespace: NAMESPACE_OF_UNHEALTHY_MACHINE
   ownerReferences:
    - apiVersion: cluster.x-k8s.io/v1alphaX
-     kind: MachineHealthCheck
-     name: REMEDIATION_GROUP
+     kind: Machine
+     name: MACHINE_NAME
 spec:
   type: reboot
-  rebootLimit: 3
+  retryLimit: 3
   timeout: 300s
```

Comment:
These need to belong to the `spec.strategy` field don't they?

Reply:
Yes, fixed!